Source/Resources/en.lproj/Localizable.strings

```diff
@@ -38,7 +38,7 @@
 "countryRegion" = "Country";
 "addressLine1" = "Address line 1";
 "addressLine2" = "Address line 2";
-"state" = "County";
+"County" = "County";
```

Comment:
I think consistency in hard to navigate files like this should be desired. All other keys are small cased so I'd just go with that. In this context `County` is not a name or title, its just another key in a file full of key-value pairs.